extensions/replication/replicationStatusProcessor/ReplicationStatusProcessor.js

```diff
@@ -92,30 +92,30 @@ class ReplicationStatusProcessor {
     /**
      * Start kafka consumer
      *
-     * @param {object} [options] - options object
-     * @param {boolean} [options.createConsumer=true] - whether to
-     *   spawn the consumer now (TEST ONLY, set to false if calling
-     *   {@link bootstrapKafkaConsumer()})
+     * @param {function} [cb] - optional callback called when startup
+     * is complete
      * @return {undefined}
      */
-    start(options) {
+    start(cb) {
         this._consumer = new BackbeatConsumer({
-            zookeeper: { connectionString: this.zkConfig.connectionString },
+            kafka: { hosts: this.kafkaConfig.hosts },
             topic: this.repConfig.replicationStatusTopic,
             groupId: this.repConfig.replicationStatusProcessor.groupId,
             concurrency:
             this.repConfig.replicationStatusProcessor.concurrency,
             queueProcessor: this.processKafkaEntry.bind(this),
             fetchMaxBytes: CONSUMER_FETCH_MAX_BYTES,
-            createConsumer: (!options || options.createConsumer),
+            bootstrap: true,
```

Comment:
Is this temporary?

Reply:
No, it's a rework of the way we bootstrap consumers for tests. Now only the bootstrap option is passed, no need to call the `bootstrap()` function later, called internally (and renamed to `_bootstrapConsumer()`).

Reply:
Oh I see what you mean, why is this true in non-test code, I have to check :)

Reply:
Fixed, this was an overlook, thanks for catching it!

Reply:
Ah I see, I wasn't too sure so decided to ask 👍